dsl/camel-jbang/camel-jbang-main/dist/CamelJBang.java

```diff
@@ -19,9 +19,9 @@
 
 //JAVA 17+
 //REPOS central=https://repo1.maven.org/maven2,apache-snapshot=https://repository.apache.org/content/groups/snapshots/
-//DEPS org.apache.camel:camel-bom:${camel.jbang.version:4.14.1}@pom
-//DEPS org.apache.camel:camel-jbang-core:${camel.jbang.version:4.14.1}
-//DEPS org.apache.camel.kamelets:camel-kamelets:${camel-kamelets.version:4.14.1}
+//DEPS org.apache.camel:camel-bom:${camel.jbang.version:4.15.0}@pom
```

Comment:
I am not sure about this, @davsclaus wdyt? IIRC we used the LTS because `jbang install camel@apache/camel` uses this file